### PR TITLE
64 returns text

### PR DESCRIPTION
**Pull request by levBagryansky**
@Graur please review 
#64 
I changed attributes that returned `string`. Now they returns `text`

### CODE REVIEW

File: src/main/eo/org/eolang/txt/text.eo [hunk 1]
```diff
@@ -36,8 +36,15 @@
...
+  [start len] > slice
+    text > @
+      ^.s.slice start len
+
   # Trim it from both sides
-  [] > trim /string
+  [] > trim
+    text > @
+      tr
+    [] > tr /string
```
Comment (Graur) [inline src/main/eo/org/eolang/txt/text.eo:47]: @levBagryansky I think we don't need to change the name of the atom, just return `EOtext()` in java code will be enough. WDYT?

File: src/main/eo/org/eolang/txt/text.eo [hunk 2]
```diff
@@ -69,25 +76,33 @@
...
-  # Returns string in lower case
-  [] > lower-case /string
-
-  # Returns string in upper case
-  [] > upper-case /string
+  # Returns text in upper case
+  [] > upper-case
+    text > @
+      uc
+    [] > uc /string
```
Comment (Graur) [inline src/main/eo/org/eolang/txt/text.eo:96]: @levBagryansky Same here

File: src/main/eo/org/eolang/txt/text.eo [hunk 3]
```diff
@@ -69,25 +76,33 @@
   #  only via EOLANG code. We can do it by using reduce method.
   [substr] > index-of /int
 
+  # Returns text in lower case
+  [] > lower-case
+    text > @
+      lc
+    [] > lc /string
```
Comment (Graur) [inline src/main/eo/org/eolang/txt/text.eo:83]: @levBagryansky Same here

File: src/main/eo/org/eolang/txt/text.eo [hunk 4]
```diff
@@ -69,25 +76,33 @@
...
   [i] > at
-    ^.s.slice i 1 > @
+    ^.slice i 1 > @
 
-  # Returns string where all substrings
+  # Returns text where all substrings
   # target changed to replacement
-  [target replacement] > replaced /string
+  [target replacement] > replaced
+    [] > replace /string
```
Comment (Graur) [inline src/main/eo/org/eolang/txt/text.eo:104]: @levBagryansky And here

**Review by Graur — CHANGES_REQUESTED** [review]
@levBagryansky Good one! Just a few suggestions:

File: src/main/eo/org/eolang/txt/text.eo (hunk 1 above)
Comment (levBagryansky) [reply src/main/eo/org/eolang/txt/text.eo:47]: @Graur agree, I'll try change to this variant

**Review by Graur — APPROVED** [review]
@levBagryansky Good one! 
@yegor256 Please, take a look

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**